File: src/main/java/net/eduard/permissions/command/PermissionsGroupSetSuffixCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.core.PermissionsGroup;
import net.eduard.permissions.core.PermissionsManager;

public class PermissionsGroupSetSuffixCommand extends CommandManager {

	public PermissionsGroupSetSuffixCommand() {
		super("setsuffix", "definirsufixo");
		setUsage("/permissions group setsuffix <group> <suffix>");
		setDescription("Definir o sufixo do grupo");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 4) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			String prefixo = Mine.toChatMessage(args[3]);
			PermissionsGroup group = manager.getGroup(nome);
			if (group != null) {
				group.prefix = prefixo;
				sender.sendMessage(EduPermissions.getInstance().message("group-set-suffix").replace("$group", "" + nome)
						.replace("$suffix", prefixo));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("group-not-exists").replace("$group", "" + nome));
			}
//				
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupSetSuffixCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.api.lib.modules.Mine
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsGroupSetSuffixCommand :
    Command("setsuffix", "definirsufixo") {
    override fun onCommand(sender: Sender, args: List<String>) {
        val manager = PermissionsAPI.instance
        if (args.size < 4) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        val suffixo = Mine.toChatMessage(args[3])
        val group = manager.getGroup(nome)
        if (group != null) {
            group.suffix = suffixo
            sender.sendMessage(
                PermsMessages.message("group-set-suffix")
                    .replace("\$group", "" + nome)
                    .replace("\$suffix", suffixo)
            )
        } else {
            sender.sendMessage(
                PermsMessages.message("group-not-exists")
                    .replace("\$group", "" + nome)
            )
        }
        //

    }


    init {
        usage = "/permissions group setsuffix <group> <suffix>"
        description = "Definir o sufixo do grupo"
    }
}